roles/build_openstack_packages/tasks/run_dlrn.yml

```diff
@@ -145,25 +145,15 @@
         - "'host' in _change"
         - "'redhat.com' in _change.host"
       block:
-        - name: Clone the downstream project
+        - name: Clone the downstream project from zuul clonned repo
           ansible.builtin.git:
-            repo: '{{ _change.host }}/gerrit/{{ _change.project }}'
+            repo: '{{ ansible_user_dir }}/src/{{ _change.host | split("://") | last }}/{{ _change.project }}'
             dest: '{{ cifmw_bop_build_repo_dir }}/DLRN/data/{{ project_name_mapped.stdout }}-tested'
-            refspec: "{{ _change.refspec }}"
-            version: 'FETCH_HEAD'
-
-        - name: Update the branch name for local DLRN clone # noqa: command-instead-of-module
-          ansible.builtin.command:
-            cmd: "git checkout -b {{ item }}"
-          args:
-            chdir: "{{ cifmw_bop_build_repo_dir }}/DLRN/data/{{ project_name_mapped.stdout }}-tested"
-          loop:
-            - "{{ cifmw_bop_osp_release }}-rhel-9-trunk"
-            - "{{ cifmw_bop_osp_release }}-trunk-patches"
+            version: "{{ cifmw_bop_osp_release }}-trunk-patches"
 
         - name: Update the project reference in packages.yml # noqa: command-instead-of-module
           vars:
-            _old_content: 'osp-patches: ssh://{{ _change.host | split("//") | last }}:22/{{ _change.project }}'
+            _old_content: 'osp-patches: git@{{ _change.host | split("//") | last }}:{{ _change.project }}.git'
```

Comment:
thecnically the git protocal is just ssh by another name but ya this makes sense